fitbenchmarking/results_output.py

```diff
@@ -173,36 +176,34 @@ def setup_neutron_VDpage_misc(function_def, results_dir):
     return support_pages_dir, details_table, see_also_link
 
 
-def setup_VDpage_misc(group_name, problem_name, res_obj, results_dir):
+def setup_VDpage_misc(group_name, problem_name, res_obj, results_dir, count):
     """
     Setup miscellaneous data for visual display page.
```

Comment:
Could you add function @param comments please? For example here not immediately obvious what count is